liveOnReboot/liveOn/View/SendGift/SendPicture/SendPictureView.swift

```diff
@@ -32,38 +30,38 @@ struct PhotoGiftView: View {
                                 Image(uiImage: image)
```

Comment:
p2;
```suggestion
                                Image(uiImage: selectedImage)
```
그냥 image라고 하기 보다는 selectedImage라고 하는 것이 더 명확할것 같습니다

Reply:
좋습니다 반영했습니다!  e45292d555de8ddabb40ed422a63140b95c02752